### PR TITLE
Feat/fly 2279

### PR DESCRIPTION
**Pull request by Hakob23**
## What
Implemented a new Management API validation for `POST /pegout/configuration` to prevent risky pegout settings where the estimated confirmation wait time could outlive quote expiration constraints.

Specifically:
- Added cross-field validation comparing:
  - estimated confirmation time from `max(rskConfirmations)` + `max(btcConfirmations)`,
  - against both `expireTime` and `expireBlocks` (converted to estimated seconds).
- Wired this validation into `SetPegoutConfigUseCase`.
- Mapped the new validation error to HTTP `400` in the pegout config handler.
- Added/updated unit tests for entity validation, use case behavior, and handler response mapping.
- Updated OpenAPI docs for `/pegout/configuration` to describe the validation and `400` response.
- Minor consistency cleanup: moved average block-time constants into their own const block.

## Why
Without this check, LPs can configure confirmations and expiry values that make quote expiration likely before required refund confirmations are reached, increasing operational risk and potential failed/refund timing issues.

## Type of Change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change 
- [x] Documentation update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Performance improvement
- [x] Test updates
- [ ] Security fix
- [ ] Deployment/Infrastructure changes

## Affected part of the project
- [x] Management UI / API
- [ ] PegIn flow
- [x] PegOut flow
- [ ] Utility scripts
- [x] Configuration files
- [ ] Metrics and alerting

## Related Issues
[Jira ticket](https://rsklabs.atlassian.net/browse/FLY-2279?atlOrigin=eyJpIjoiMzMwNmY4ZDBmNTA4NDBmYTlhNjBkZDBkZDgwMTZmNDYiLCJwIjoiaiJ9)

## How to test
1. Run unit tests:
   - `go test ./internal/entities/liquidity_provider ./internal/usecases/liquidity_provider ./internal/adapters/entrypoints/rest/handlers`
2. Management API validation checks:
   - Send `POST /pegout/configuration` with values where estimated confirmation time is **below** both `expireTime` and `expireBlocks` window → expect `204`.
   - Send `POST /pegout/configuration` with values where estimated confirmation time is **>= expireTime** → expect `400` with validation error details.
   - Send `POST /pegout/configuration` with values where estimated confirmation time is **>= expireBlocks * AvgRskBlockTimeSec** → expect `400` with validation error details.
3. Regression checks:
   - Existing bridge min and non-positive wei validations still return `400`.
   - Non-validation failures still return `500`.

## Reviewer Guidelines
- Focus on the validation formula and boundary condition (`>=` is intentional).
- Verify fallback behavior when general config is unavailable (defaults are used).
- Confirm error mapping in handler returns `400` only for validation-class errors.
- Check test coverage for:
  - pass case,
  - expireTime violation,
  - expireBlocks violation,
  - fallback path (nil/error general config).

### CODE REVIEW

**Review by TravellerOnTheRun — COMMENTED** [review]
## Blocker — security or funds risk

- **The LPS E2E check “Server initialization and health check” failed on this PR head commit.** — Mandatory gate per review policy; failure is CI/environment (missing LBC checkout), not in the pegout validation diff.

## Required before merge

None.

## Recommended

- **FLY-2279 describes deposit and transfer confirmation counts, but this PR validates using the maximum RSK and BTC confirmation requirements from general LP config — please confirm with product that this substitution matches the ticket intent.** — AC-5 Partial: implementation and fallback tests exist, but ticket terminology differs from code semantics.

- **The OpenAPI description says deposit and transfer confirmations, but the validation uses maximum RSK and BTC confirmation counts from general LP configuration — update the spec text to match the implementation.** — see inline suggestion on `OpenApi.yml` ~L1306

## Optional nit

- **Expiry validation uses fixed average block times (30s RSK, 600s BTC) rather than live chain parameters; confirm these constants are acceptable for Management API configuration checks.** — Product/ops judgment on fixed constants, not a code defect on this PR.

- **Add a handler test that asserts HTTP 400 when the expireBlocks-derived window is too short, mirroring the entity and use-case coverage for that branch.** — AC-7 coverage gap at HTTP boundary only; entity and use-case tests cover the branch.

File: OpenApi.yml [hunk 1]
```diff
@@ -1304,14 +1304,18 @@ paths:
   /pegout/configuration:
     post:
       description: ' Set the configuration for the Pegout service. Included in the
```
Comment (TravellerOnTheRun) [inline OpenApi.yml:1402]: The OpenAPI description says deposit and transfer confirmations, but the validation uses maximum RSK and BTC confirmation counts from general LP configuration — update the spec text to match the implementation.

```suggestion
      description: ' Set the configuration for the Pegout service. Included in the
        management API. The configuration is rejected when the estimated confirmation
        wait (from max RSK and BTC confirmations in general LP configuration) is greater
        than or equal to expireTime or to the time implied by expireBlocks.'
```

**Review by TravellerOnTheRun — APPROVED** [review]
